src/app/models/StorageTypes.ts

```diff
@@ -36,6 +36,8 @@ export interface ApiNetwork {
 	genesisTs: number
 	elCurrency: NetworkMainCurrency
 	clCurrency: NetworkMainCurrency
+	slotPerEpoch: number
```

Comment:
**Bug:** You will also have to use this here:
https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/beb653097d99f32278e8c80ccc5c246904d97a85/src/app/components/dashboard/dashboard.component.ts#L220
https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/f96f10bddcb148b44dbc929c8ef8369c2c58f652/src/app/controllers/OverviewController.ts#L838
https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/f96f10bddcb148b44dbc929c8ef8369c2c58f652/src/app/controllers/OverviewController.ts#L891-L894